src/main/java/net/masterthought/cucumber/ReportBuilder.java

```diff
@@ -83,6 +84,9 @@ public Reportable generateReports() {
             // first copy static resources so ErrorPage is displayed properly
             copyStaticResources();
 
+            // copy custom js and css resources if specific in configuration
+            copyCustomJsAndCssResources();
```

Comment:
I will be very happy if you extend https://github.com/damianszczepanik/cucumber-reporting/blob/master/src/test/java/LiveDemoTest.java with some fancy css style for any item you like so demo http://damianszczepanik.github.io/cucumber-html-reports/report-feature_1920820787.html presents feature you want to deliver by this change

Reply:
I use this to clean up the "Output" from after hooks (you might think directly logging "good" output would be better, but this is another discussion...) and for example to add syntax highlighting to logged requests.

I could add syntax highlighting to the xml request in the live demo, however, for this I would need to include a 3rd party library for example highlightjs. If that is ok for you, I will add that.

Otherwise I could just "randomly" change the styling a bit, but as I said, I do not really use it for that purpose...

Reply:
Highlights seems fine as well - the goal is to show how this can be used and what are the results